lib/driver.js

```diff
@@ -1075,6 +1075,15 @@ class XCUITestDriver extends BaseDriver {
     return _.toLower(this.opts.platformName) === _.toLower(PLATFORM_NAME_TVOS);
   }
 
+  checkCondition () {
```

Comment:
Please move this method to `condition.js` and rename it to `requireConditionInducerCompatibleDevice`